CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ The sections should follow the order `Added`, `Changed`, `Fixed`, and `Removed`.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Changed
+
+- **Breaking** fontconfig system library provider changed to yeslogic-fontcontconfig-sys
+- **Breaking** freetype-rs bumped to 0.36.0
```

Comment:
I'm not sure how to better communicate that it should cascade breaking bump every rust crate.